packages/keystone/src/admin-ui/system/createAdminMeta.ts

```diff
@@ -98,6 +108,8 @@ export function createAdminMeta(
     }
 
     for (const [fieldKey, field] of Object.entries(list.fields)) {
+      // If the field is a relationship field and is related to an omitted list, skip.
+      if (field.dbField.kind === 'relation' && omittedLists.includes(field.dbField.list)) continue;
```

Comment:
We want to check if the field is a relationship to an omitted list, is interrogating the dbField on said list the right way to make this check? @timleslie

Reply:
Yep